ops/testing.py

```diff
@@ -2126,6 +2159,8 @@ def __init__(self, unit_name: str, meta: charm.CharmMeta, config: '_RawConfig'):
         self._networks: Dict[Tuple[Optional[str], Optional[int]], _NetworkDict] = {}
         self._reboot_count = 0
         self._running_action: Optional[_RunningAction] = None
+        # For `Model.get_cloud_spec`, initialised to None.
```

Comment:
I think this is pretty obvious, and "initialized to None" is what `= None` does, so IMO we can just remove this comment.